contrib/ansible/openshift/register.sh

```diff
@@ -87,4 +93,8 @@ fi
 if [ $? -ne 0 ]; then
   warn_log "cannot delete hosts file" 
 fi
+/bin/rm /tmp/prometheus_metrics_status
```

Comment:
The removal has to be done with

      trap "rm -f /tmp/prometheus_metrics_status" EXIT QUIT INT

early in the script (before the file is created), so that the file goes away on abnormal termination, as well as on normal exit.